lms/djangoapps/course_api/tests/test_serializers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Test data created by CourseSerializer
+"""
+
+from datetime import datetime
+
+from rest_framework.test import APIRequestFactory
+from rest_framework.request import Request
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.course_module import DEFAULT_START_DATE
+
+from ..serializers import CourseSerializer
+from .mixins import CourseApiFactoryMixin
+
+
+class TestCourseSerializerFields(CourseApiFactoryMixin, ModuleStoreTestCase):
```

Comment:
OK, makes sense.

TBH I actually think it might be possible to create all your test courses in `setUpClass` using `SharedModuleStoreTestCase` since they aren't actually mutated during tests; however, I would call this a purely optional optimization whose gains might not be worth the trouble - so never mind.